elements/pl-checkbox/pl-checkbox.py

```diff
@@ -481,8 +491,9 @@ def test(element_html, data):
                 raise ValueError(f'Unknown value for partial_credit_method: {partial_credit_method}')
         else:
             score = 0
+        feedback = [option['feedback'] for option in data['params'][name] if option['key'] in ans]
```

Comment:
Let's use `.get()` here too for safety.

```suggestion
        feedback = [option.get('feedback', None) for option in data['params'][name] if option['key'] in ans]
```